plugins/inputs/linux_fscache/linux_fscache.go

```diff
@@ -0,0 +1,67 @@
+package linux_fscache
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"strconv"
+	"strings"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/plugins/inputs"
+)
+
+type FSCache struct {
+	statFile string
+}
+
+func (f *FSCache) Description() string {
+	return "Get FS-Cache statistics from /proc/fs/fscache/stats"
+}
+
+func (f *FSCache) SampleConfig() string { return "" }
+
+func (f *FSCache) Gather(acc telegraf.Accumulator) error {
+
+	s, err := os.Stat(f.statFile)
+	if err != nil {
+		return err
+	} else if s.Size() == 0 {
+		return fmt.Errorf("fscache: %s zero length!", f.statFile)
+	}
+
+	data, err := ioutil.ReadFile(f.statFile)
+	if err != nil {
+		return err
+	}
+
+	lines := strings.Split(string(data), "\n")
+	fields := make(map[string]interface{})
+	for i := 1; i < len(lines); i++ {
```

Comment:
Use `for _, line := range lines[1:] {` or (preferably) `bufio.Scanner`.  If you use the first form, make sure you check the length of lines to avoid a out of bounds panic.

Reply:
done.